app/src/main/res/drawable/ic_anim_favorite_checking.xml

```diff
@@ -0,0 +1,101 @@
+<animated-vector xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    tools:ignore="NewApi"
+    >
+    <aapt:attr name="android:drawable">
+        <vector
+            android:name="vector"
+            android:width="48dp"
+            android:height="48dp"
+            android:viewportWidth="48"
+            android:viewportHeight="48">
+            <group android:name="page_1">
+                <group
+                    android:name="artboard"
+                    android:pivotX="24"
+                    android:pivotY="24">
+                    <path
+                        android:name=""
```

Comment:
Why `android:name=""`  need? 📛

Reply:
Oh, I made this file with https://shapeshifter.design/ and didn't check the content of xml... Sorry.
It is clear that `android:name="artboard"` is useless. I will remove these lines.